workers/ohsome_quality_analyst/geodatabase/client.py

```diff
@@ -143,51 +143,29 @@ async def get_area_of_bpolys(bpolys: Dict):
                     )
             ) / (1000*1000) as area_sqkm
         """
-    polygon = json.dumps(bpolys["features"][0]["geometry"])
     async with get_connection() as conn:
-        result = await conn.fetchrow(query, polygon)
+        result = await conn.fetchrow(query, geojson.dumps(bpolys))
```

Comment:
```suggestion
        # TODO when fixing the TODO to allow geojson.Feature: make sure the geometries (e.g. Polygon) are returned
        result = await conn.fetchrow(query, geojson.dumps(bpolys))
```

Reply:
Not sure what you mean here. Do you expect that the function takes a Feature instead of Geomety as input?
What do you mean be returning the geometry in this case?

Reply:
That's what you (or someone else) put as a TODO above the function.
`# TODO Rewrite to work with geojson.Feature as input`

Reply:
Ah yes, that was probably me. But I changed my mind on that. I think just the geometry is more appropriate.

Reply:
then maybe delete the other TODO ;)

Reply:
Sorry, I deleted the TODO comment